### PR TITLE
feat(policy): add docs for open telemetry in meshmetric policy

### PR DESCRIPTION
**Pull request by Automaat**
Add docs for OpenTelemtry in MeshMetric

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits)

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: app/_src/policies/meshmetric.md [hunk 1]
```diff
@@ -231,6 +238,29 @@ scrape_configs:
...
+#### OpenTelemetry (experimental)
+
+```yaml
+backends:
+  - type: OpenTelemetry
+    openTelemetry: 
+      endpoint: http://otel-collector.observability.svc:4317
+```
+
+This configuration tells {{site.mesh_product_name}} Dataplane Proxy to push metrics to [OpenTelemetry collector](https://opentelemetry.io/docs/collector/).
```
Comment (jakubdyszkiewicz) [inline app/_src/policies/meshmetric.md:250]: nit: we use "data plane proxy" (lowercase, separate words)

File: app/_src/policies/meshmetric.md [hunk 2]
```diff
@@ -231,6 +238,29 @@ scrape_configs:
...
+Dataplane Proxy will scrape metrics from Envoy and other [applications](/docs/{{ page.version }}/policies/meshmetric/#applications) 
+and push them to configured OpenTelemetry collector.
```
Comment (jakubdyszkiewicz) [inline app/_src/policies/meshmetric.md:252]: nit: and other applications in a Pod/VM.

to emphasise that it's not random applications in kube clusters

File: app/_src/policies/meshmetric.md [hunk 3]
```diff
@@ -231,6 +238,29 @@ scrape_configs:
...
+
+When you configure application scraping make sure to specify `application.name` to utilize [OpenTelemetry scoping](https://opentelemetry.io/docs/concepts/instrumentation-scope/) 
+
+##### Limitations
+
+- You cannot configure scraping interval for OpenTelemetry. By default, it will publish metrics to collector every **30 seconds**. Ability to configure scraping interval in policy will be added in next release.
```
Comment (jakubdyszkiewicz) [inline app/_src/policies/meshmetric.md:258]: nit: in the future. I think we should avoid committing to specific releases in docs.

**Review by jakubdyszkiewicz — APPROVED** [review]

File: app/_src/policies/meshmetric.md (hunk 1 above)
Comment (Automaat) [reply app/_src/policies/meshmetric.md]: changed

File: app/_src/policies/meshmetric.md (hunk 2 above)
Comment (Automaat) [reply app/_src/policies/meshmetric.md:252]: changed

File: app/_src/policies/meshmetric.md (hunk 3 above)
Comment (Automaat) [reply app/_src/policies/meshmetric.md]: changed

**Pull request merged**